### DISCUSSION TITLE
Postman Shared Workspace

### DISCUSSION THREAD

**Discussion by yehandesilva** [yehandesilva/monkeyPoll#25] [General]
We can use a shared postman workspace to save individually typing out each request. Whoever is adding a new route/controller can create and organize new requests making our API call management easier.

[Invite code](https://app.getpostman.com/join-team?invite_code=8e8329dc990627cd06d62deaa07f90c2&target_code=0fd5f83cb44706c4ffca5480db39e21f)

**Comment by yehandesilva** [comment]
@AminZeina If you can use this when developing the user controller, that would be great.

**Reply by AminZeina** [reply]
Thats a great idea, looks good. Will do!

**Reply by AminZeina** [reply]
@yehandesilva could you resend the invite code, it's expired

**Reply by yehandesilva** [reply]
[New code.](https://app.getpostman.com/join-team?invite_code=db4f68485f492b18e2c4680103bea635&target_code=83b3bca0c8b13fb257a86c06cca9c92a) This workspace is public since only up to 3 users can have a private workspace.

**Reply by AminZeina** [reply]
Perfect thanks, I've sent a request to join the workspace
